From 241105311073cbf8a1fceea8c6c3b6485a4d128b Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Thu, 13 May 2021 11:29:01 +0100
Subject: [PATCH] rockchip/rk3399: fix dram section placement declaration

In commit af27fb89a9 the DRAM restore code was moved to PMUSRAM but
the declaration in dram.h wasn't updated to reflect this change. Newer
versions of gcc are now complaining about this with certain flags.

Details in https://developer.trustedfirmware.org/T925

Fixes: af27fb89a9 ("rockchip/rk3399: Move DRAM restore to PMUSRAM")
Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
Signed-off-by: Chris Kay <chris.kay@arm.com>
Change-Id: Ia96134465892989821ef90572203b2755003f7fb
---

diff --git a/plat/rockchip/rk3399/drivers/dram/dram.h b/plat/rockchip/rk3399/drivers/dram/dram.h
index 0eb12cf..5572b16 100644
--- a/plat/rockchip/rk3399/drivers/dram/dram.h
+++ b/plat/rockchip/rk3399/drivers/dram/dram.h
@@ -149,7 +149,7 @@
 	uint32_t rx_cal_dqs[2][4];
 };
 
-extern __sramdata struct rk3399_sdram_params sdram_config;
+extern struct rk3399_sdram_params sdram_config;
 
 void dram_init(void);
 
